src/streamsync/ui_manager.py

```diff
@@ -30,102 +24,119 @@ def assert_in_container():
         if container is None:
             raise UIError("A component can only be created inside a container")
 
+    @property
+    def component_tree(self) -> ComponentTree:
```

Comment:
I see that this property is not used, and other methods still access `current_component_tree()` directly. I believe we could replace all calls to it with this property, or is there something I am missing?

Reply:
I keep it for debug purpose on the IDE.

Reply:
Finally we can't because the method are static and component tree is a property. It's not accessible